### PR TITLE
Improved error handling of re-numbering motions of a category.

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: openslides/motions/views.py [hunk 1]
```diff
@@ -618,6 +618,8 @@ def numbering(self, request, pk=None):
 
                 # Set new identifers and change identifiers of amendments.
                 for obj in motions_to_be_sorted:
+                    if Motion.objects.filter(identifier = obj['new_identifier']).exists():
+                        raise ValidationError({'detail': _('Numbering aborted because the motion identifier "%s" exists already outside of this category.') % obj['new_identifier']})
```
Comment (FinnStutzenstein) [inline openslides/motions/views.py]: Typo: 'already exists'

File: openslides/motions/views.py [hunk 2]
```diff
@@ -618,6 +618,8 @@ def numbering(self, request, pk=None):
 
                 # Set new identifers and change identifiers of amendments.
                 for obj in motions_to_be_sorted:
+                    if Motion.objects.filter(identifier = obj['new_identifier']).exists():
```
Comment (FinnStutzenstein) [inline openslides/motions/views.py]: Please remove the whitespaces around the `=`.

Comment (FinnStutzenstein) [reply openslides/motions/views.py]: (The only travis error)

File: openslides/motions/views.py (hunk 1 above)
Comment (emanuelschuetze) [reply openslides/motions/views.py]: done

File: openslides/motions/views.py (hunk 2 above)
Comment (emanuelschuetze) [reply openslides/motions/views.py]: done

**Comment by emanuelschuetze** [comment]
Travis fixed. - I have to merge it now.
@normanjaeckel @FinnStutzenstein Please review later.

**Pull request merged**